src/python/py/models/builder.py

```diff
@@ -291,6 +291,7 @@ def __init__(self, config, io_dtype, onnx_dtype, ep, cache_dir, extra_options):
                 "block_size": int(extra_options.get("int4_block_size", 32)),
                 "is_symmetric": extra_options.get("int4_is_symmetric", True),
                 "op_types_to_quantize": extra_options.get("int4_op_types_to_quantize", ("MatMul", )),
+                "exclude_nodes": extra_options.get("int4_exclude_nodes", []),  # List of nodes to exclude from quantization
```

Comment:
To match how the other attributes use the same names as the constructor's argument names

```suggestion
                "nodes_to_exclude": extra_options.get("int4_nodes_to_exclude", []),  # List of nodes to exclude from quantization
```